README.md

```diff
@@ -197,10 +197,23 @@ the
 [Dynatrace documentation](https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/enrich-metrics/).
 By default, this option is turned on.
 
-## Histogram
+## Limitations
+
+### Histogram
 
 OpenTelemetry Histograms are exported to Dynatrace as statistical summaries
 consisting of a minimum and maximum value, the total sum of all values, and the
 count of the values summarized. If the min and max values are not directly
 available on the metric data point, estimations based on the boundaries of the
 first and last buckets containing values are used.
+
+### Typed attributes support
+
+The OpenTelemetry Metrics API for JavaScript supports the concept of Attributes.
+These attributes consist of key-value pairs, where the keys are strings and the
+values are either `string`, `bool`, or `number` or arrays of the previously
+mentioned types and also including `null` and `undefined`.
+
+At the moment, this exporter only supports attributes with string key and value
+type. This means that if attributes of any other type are used, they will be
+ignored and only the string-valued attributes are going to be sent to Dynatrace.
```

Comment:
Since we actually don't support typed attributes the name of this section is a little misleading. I also don't think we need to re-explain attributes. It is probably sufficient to just link to the spec definition? Understanding attribute isn't really necessary for use of the exporter.

```suggestion
### Attribute type limitations

Currently, only `string` type attribute values are supported.
Attributes with values of any other type will be dropped and not exported.
If you need those values to be exported, please convert them to `string` type before export.
```